### PR TITLE
fix : city page fix

### PR DESCRIPTION
**Pull request by hibabdelk**

### CODE REVIEW

File: src/Pages/CitiesPage/CitiesPage.js [hunk 1]
```diff
@@ -1,18 +1,21 @@
 import React from 'react';
-import { citiesData } from './citiesData';
-import CityItem from '../../Components/CityItem/CityItem.js';
+import CITIES from '../../MockData/cities.json';
```
Comment (osamaakb) [inline src/Pages/CitiesPage/CitiesPage.js]: this should be small case
```suggestion
import cities from '../../MockData/cities.json';
```

**Review by osamaakb — CHANGES_REQUESTED** [review]

**Comment by osamaakb** [comment]
can I get more info about this PR?

**Comment by hibabdelk** [comment]
Since we changed the city card design so sat the page design as grid view

**Comment by osamaakb** [comment]
ok, can you please just change the `CITIES` to `cities` then

**Review by osamaakb — APPROVED** [review]

**Review by osamaakb — APPROVED** [review]

**Pull request merged**